### ISSUE TITLE
SSL certificate problem: unable to get local issuer certificate

### ISSUE THREAD

**Issue by jeroen** [r-windows/checks#76]
See `robotstxt` and `rzeit2` or `Rlabkey`

**Comment by jeroen** [comment]
Note that curl itself has:

```
  This is libcurl version 7.64.1 with OpenSSL/1.1.1a (Schannel) 
  Using test server: http://hb.opencpu.org 
  -- 1. Failure: Invalid domain raises an error (@test-certificates.R#15)  -------
  `curl_fetch_memory(fake_url)` did not throw an error.
  
  == testthat results  ===========================================================
  [ OK: 216 | SKIPPED: 1 | WARNINGS: 0 | FAILED: 1 ]
  1. Failure: Invalid domain raises an error (@test-certificates.R#15) 
```

**Comment by jeroen** [comment]
Uwe found that the problem is that the [standard build script](https://github.com/wch/r-source/blob/06185bd431beb9b6e0883193273d1779ef2c6a2c/src/gnuwin32/Makefile#L146-L148) assumes the cert bundle can be copied from the $(CURL_PATH) folder:

```
ifdef USE_LIBCURL
	@if test -f "$(CURL_PATH)/etc/curl-ca-bundle.crt"; then \
	  $(CP) -p "$(CURL_PATH)/etc/curl-ca-bundle.crt" ../../etc; \
	fi
endif
```

However we don't set `CURL_PATH` anymore.

**Comment by jeroen** [comment]
The results for `robotstxt` have not yet been updated on CRAN, but if I submit to winbuilder it seems OK.

**Comment by jeroen** [comment]
This was fixed on Winbuilder by installing the cert bundle in the right place.
